### ISSUE TITLE
Number of Multi-Bar Rest not spoken by Screen Reader

### ISSUE THREAD

**Issue by jmwmusic** [musescore/MuseScore#33788]
### Issue type

Accessibility issue (e.g. for keyboard-only or screen reader users)

### Description with steps to reproduce

When multi-bar rests are being used the screen reader does not state how many bars the rest is, it just says "multi-bar rest bar" followed by the bar/beat of where it begins.  The number of bars is not spoken.

STEPS:
1.  Create a score with some bars of notes and some empty bars at least 2 empty bars in a row. 
2. Press Ctrl + Shift + M to Toggle Multi-bar rests on.
3. With a screen reader such as NVDA enabled use arrow keys to move on to the multi-bar rest and notice it does not speak the number of bars.

### Supporting files, videos and screenshots

N/A

### In which versions of MuseScore Studio is this issue present?

4.7.2

### Regression

I was unable to check

### Operating system

Windows 11 with NVDA

### Checklist

- [x] This report follows the [guidelines](https://github.com/musescore/MuseScore/wiki/Reporting-bugs-and-issues) for reporting bugs and issues
- [x] I have verified that this issue has not been logged before, by searching the [issue tracker](https://github.com/musescore/MuseScore/issues) for similar issues
- [x] I have attached all requested files and information to this report
- [x] I have attempted to identify the root problem as concisely as possible, and have used minimal reproducible examples where possible
